### ISSUE TITLE
Safety goggles says "block all *self-inflicted* damage this turn", but due to how the card works, it also blocks thorns damage

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeonsbeta#757]
Maybe reword it to something like "block all damage this turn (but not the enemy's turn)"?

**Comment by TerryCavanagh** [comment]
Oh, hmm - I can see the confusion, but I think it's reasonable to read thorns damage as "self-inflicted". Also, I've already gotten it translated :D Gonna leave this one as is, I think.
